enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -99,11 +99,11 @@ export const _InteractiveQuestion = ({
 
   useEffect(() => {
     if (queryResults) {
-      setLoading(false);
+      setIsQuestionLoading(false);
     }
   }, [queryResults]);
 
-  if (loading) {
+  if (isQuestionLoading || isRunning) {
```

Comment:
here's the key fix. We should display loading sprinner when either
1. the question is loading
2. the query is running